pkg/storage-init/storage-init.sh

```diff
@@ -148,6 +153,7 @@ if P3=$(findfs PARTLABEL=P3) && [ -n "$P3" ]; then
         fi
     fi
 
+    echo -ne "$P3_FS_TYPE" > /run/eve.persist_type
```

Comment:
If mkfs fails above on line 136 we echo some random bytes.

Reply:
> If mkfs fails above on line 136 we echo some random bytes.
P3_FS_TYPE is only modified on basis of mkfs.ext4 result, and P3_FS_TYPE is initialised at line 109. So looks fine to me. 
Am I missing something @eriknordmark ?

Reply:
> > If mkfs fails above on line 136 we echo some random bytes.
> > P3_FS_TYPE is only modified on basis of mkfs.ext4 result, and P3_FS_TYPE is initialised at line 109. So looks fine to me.
> > Am I missing something @eriknordmark ?

ok, got it. We should probably set P3_FS_TYPE to unknown in case of mkfs.ext4 failure in 136

Reply:
Hey @nordmark -- why are you saying that these would be random? They are just whatever came back from blkid -- that's not really random.

I'd rather use that than unknown @cshari-zededa

Reply:
@rvs if blkid says ext4 but fsck has a failure and as a result we try to mkfs, but mkfs fails, should we really proceed as if it is a good ext4 filesystem? I think that is what the current code does. This addition merely saves the ext4 in /run/eve but that propagates the lie that it is a usable filesystem.